CHANGELOG.md

```diff
@@ -6,8 +6,7 @@ See README.md on how to build the hipCUB documentation using Doxygen.
 ### Addded
 - Initial HIP on Windows support. See README for instructions on how to build and install.
 ### Changed
-- Packaging split into a runtime package called hipcub and a development package called hipcub-devel. The development package depends on runtime. The runtime package suggests the development package for all supported OSes except CentOS 7 to aid in the transition. The suggests feature in packaging is introduced as a deprecated feature and will be removed in a future rocm release.
-    - As hipCUB is a header-only library, the runtime package is an empty placeholder used to aid in the transition. This package is also a deprecated feature and will be removed in a future rocm release.
+- Packaging changed to a development package (called hipcub-dev for Debian, and hipcub-devel for other OSes). As hipCUB is a header-only library, there is no runtime package. To aid in the transition, the development package sets the "provides" field to provide the package hipcub, so that existing packages depending on hipcub can continue to work. This provides feature is introduced as a deprecated feature and will be removed in a future ROCm release.
```

Comment:
for hipcub-dev for debian packaging based OSes and hipcub-devel for rpm packaging based OSes.

Reply:
something like that because it is related to the package type rather than OS.

Reply:
Updated to use "`.deb`" packages instead of "Debian", and "`.rpm`" packages instead of "other OSes".